Plausible/Chamelean/DeriveSchedules.lean

```diff
@@ -473,6 +509,59 @@ private partial def enum_schedules {α v} [BEq v] (vars : List v) (hyps : List (
 #guard_msgs(error, drop info) in
 #eval (enum_schedules [`n, `m] [(`n_le_m, [], [`n, `m])] [`n,`m]).take 5
 
+partial def enum_schedules' {α v} [BEq v] (vars : List v) (hypComps : List (List (α × List (List v) × List v))) (env : List v)
```

Comment:
What does this function do, and how does it relate to `enum_schedules`?

Aside: Why aren't we using `camelCase`?